cassandra-analytics-core/src/test/java/org/apache/cassandra/spark/TestUtils.java

```diff
@@ -278,6 +278,14 @@ public static Gen<CassandraVersion> versions()
         return arbitrary().pick(CassandraVersion.implementedVersions());
     }
 
+    public static Gen<CassandraVersion> versions(CassandraBridge bridge)
+    {
+        List<CassandraVersion> versions = Arrays.stream(CassandraVersion.implementedVersions())
+                                                .filter(v -> bridge.getVersion().equals(v))
+                                                .collect(Collectors.toList());
+        return arbitrary().pick(versions);
+    }
```

Comment:
hmm.. what is the point to having the `Gen` here? It will only produce one and only one version.

Reply:
When I was implementing it, I though we could have multiple testable versions with same bridge. But C* version is major + minor (e.g. 40, 41, 50), so indeed this does not make sense.